tests/epyccel/modules/numpy_sign.py

```diff
@@ -0,0 +1,177 @@
+# pylint: disable=missing-function-docstring, missing-module-docstring\
+
+from pyccel.decorators import types
+
+def complex_pos():
+    import numpy as np
+    b = np.sign(complex(1+2j))
+    return b
+
+def complex_neg():
+    import numpy as np
+    b = np.sign(complex(-1-2j))
+    return b
+
+def complex64_pos():
+    import numpy as np
+    b = np.sign(np.complex64(64+64j))
+    return b
+
+def complex64_neg():
+    import numpy as np
+    b = np.sign(np.complex64(-64-64j))
+    return b
+
+def complex128_pos():
+    import numpy as np
+    b = np.sign(np.complex128(128+128j))
+    return b
+
+def complex128_neg():
+    import numpy as np
+    b = np.sign(np.complex128(-128-128j))
+    return b
```

Comment:
You have not tested the case where the real part is positive and the imaginary part is negative, or vice versa. I suspect that you may be up for a surprise!

Reply:
Fixed in the last commit